policy-controller/grpc/src/lib.rs

```diff
@@ -370,9 +370,14 @@ fn to_http_route_list<'r>(
     // namespace.
     let mut route_list = routes.into_iter().collect::<Vec<_>>();
     route_list.sort_by(|(a_name, a), (b_name, b)| {
```

Comment:
tiny nit, but this are route refs now, rather than names.

Reply:
done https://github.com/linkerd/linkerd2/pull/9036/commits/23f9ec82d3e18c0fbecf6ceef84575cd0ff5e8c8